x/budget/keeper/budget.go

```diff
@@ -7,8 +7,9 @@ import (
 	"github.com/tendermint/budget/x/budget/types"
 )
 
+// BudgetCollection collets all the valid budgets registered in params.Budgets and
+// distributes the total collected coins to collection address.
```

Comment:
Can we rename `BudgetCollection` to `CollectBudgets`? @dongsam @kogisin

Reply:
I prefer `CollectBudgets` as well

Reply:
I agree, please update with spec and relevant docs

Reply:
Also I just spot a typo in the comment, `collets` -> `collects`

Reply:
I'll make the changes now